week2/homework/src/index.js

```diff
@@ -1,3 +1,82 @@
 'use strict';
+const fs = require("fs");
+const json = fs.readFileSync("./tasks.json", "utf-8");
+const jsonData = JSON.parse(json);
+let command = process.argv[2];
+let value = process.argv[3];
+let index = parseInt(value) - 1;
+let updateTask = process.argv[4];
+
+switch (command) {
+  case 'list':
+    listTasks();
+    break;
+  case 'add':
+    addTask(value);
+    break;
+  case 'remove':
+    removeTask(index);
+    break;
+  case 'reset':
+    restTasks();
+    break;
+  case 'update':
+    updateTasks(index, updateTask);
+    break;
+  case 'help':
+  default:
+    fs.readFile('./help.txt', 'utf8', function (error, data) {
+      if (error) {
+        console.error(error);
+      } else {
+        console.log(data);
+      }
+    });
+}
+
+function listTasks() {
+  if (jsonData.length > 0) {
```

Comment:
you can reduce the code needed by using [Array.forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)
```js
function listTasks() {
  jsonData.forEach(el => console.log(el.task));
}
```